apps/rehabstod/src/pages/CurrentSickLeaves/components/TableBodyRows.test.tsx

```diff
@@ -67,15 +67,15 @@ describe('Navigate', () => {
                 <TableBodyRows
                   isLoading={false}
                   showPersonalInformation
-                  sickLeaves={[fakerFromSchema(sickLeaveInfoSchema)({ patient: { id: 'aperiam' } })]}
+                  sickLeaves={[fakerFromSchema(sickLeaveInfoSchema)({ uid: 'aperiam' })]}
                   unitId="Alfa Vårdenhet"
                   isDoctor={false}
                 />
               </tbody>
             </Table>
           }
         />
-        <Route path={`/pagaende-sjukfall/${btoa('aperiam')}`} element={<p>Patient Route</p>} />
+        <Route path="/pagaende-sjukfall/aperiam" element={<p>Patient Route</p>} />
```

Comment:
Is this correct?

Reply:
I believe so. From what I can tell we previously set 'aperiam' to patient.id. This has been updated to uid instead. Since the uid is already encrypted, there is no need to Base64.encode the string?

Reply:
Ah sorry I didn't see it was a test. I thought you were hard coding a url.